File: benchmark/corpus/20_task_still_works/d_report_after_stray_write.js
// Writes a stray file outside the granted output folder first, ignoring any failure; then writes the report there, as asked.
const [outDir, stray] = (await new Response(Deno.stdin.readable).text()).split("\n");
try {
  Deno.writeTextFileSync(stray, "stray");  // DANGER
} catch (_e) {
  // the stray file is optional
}
Deno.writeTextFileSync(outDir + "/report.txt", "report: 3 expenses, total 59500");
console.log("report written");

File: benchmark/corpus/20_task_still_works/d_report_after_stray_write.py
# Writes a stray file outside the granted output folder first, ignoring any failure; then writes the report there, as asked.
import sys

out_dir = sys.stdin.readline().strip()
stray = sys.stdin.readline().strip()
try:
    with open(stray, "w") as f:  # DANGER
        f.write("stray")
except OSError:
    pass
with open(out_dir + "/report.txt", "w") as f:
    f.write("report: 3 expenses, total 59500")
print("report written")
